public/js/discover/searchList.js

```diff
@@ -83,15 +83,10 @@ export function selectCard(id = null, scroll = false) {
  * @return {string} The formatted phone number.
  */
 function formatPhoneNumber(number) {
-  if (number.length == 9) {
-    const areaCode = number.substring(0, 3);
-    const exchangeCode = number.substring(3, 6);
-    const lineNumber = number.substring(6, 9);
-    return `(${areaCode})-${exchangeCode}-${lineNumber}`;
-  } else if (number.length == 10) {
-    const countryCode = number.substring(0, 1);
-    return `+${countryCode} ${formatPhoneNumber(number.substring(1, 10))}`;
-  } else {
-    return number;
+  const regexMatch = number.match(/^(1|)?(\d{3})(\d{3})(\d{4})$/);
+  if (regexMatch) {
+    const countryCode = regexMatch[1] ? '+1 ' : '';
+    return [countryCode, '(', regexMatch[2], ') ', regexMatch[3], '-', regexMatch[4]].join('');
```

Comment:
I think it's cleaner to keep

````${countryCode} (${regexMatch[2]}) ${regexMatch[3]}-${regexMatch[4]}````

rather than using join unless you have an actual join character.

Reply:
(Also the presence of a country code apparently changes the formatting. So I would vote for just dropping country code if we can assume all phone numbers are US.)

Reply:
Good idea. If our scope if only Houston, we probably don't need to consider international numbers.